From fb17611b517a54ff07ccb518bd45fc6f321c9b91 Mon Sep 17 00:00:00 2001
From: Kai Meng <quic_kmeng@quicinc.com>
Date: Wed, 27 Jul 2022 16:58:54 +0800
Subject: [PATCH] refpolicy: update booleans

1.Allow users to resolve user passwd entries directly
from ldap rather then using a sssd server
2.Allow the mount command to mount any directory or file
3.Enable support for systemd-tmpfiles to manage all
non-security files

Upstream-Status: Pending

Signed-off-by: Kai Meng <quic_kmeng@quicinc.com>
---
 policy/modules/system/authlogin.te | 2 +-
 policy/modules/system/mount.te     | 2 +-
 policy/modules/system/systemd.te   | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/policy/modules/system/authlogin.te b/policy/modules/system/authlogin.te
index 345e07f..23fddf8 100644
--- a/policy/modules/system/authlogin.te
+++ b/policy/modules/system/authlogin.te
@@ -11,7 +11,7 @@ policy_module(authlogin, 2.14.0)
 ## Allow users to resolve user passwd entries directly from ldap rather then using a sssd server
 ## </p>
 ## </desc>
-gen_tunable(authlogin_nsswitch_use_ldap, false)
+gen_tunable(authlogin_nsswitch_use_ldap, true)
 
 attribute can_read_shadow_passwords;
 attribute can_write_shadow_passwords;
diff --git a/policy/modules/system/mount.te b/policy/modules/system/mount.te
index 9d8479d..05d0d0b 100644
--- a/policy/modules/system/mount.te
+++ b/policy/modules/system/mount.te
@@ -10,7 +10,7 @@ policy_module(mount, 1.20.0)
 ## Allow the mount command to mount any directory or file.
 ## </p>
 ## </desc>
-gen_tunable(allow_mount_anyfile, false)
+gen_tunable(allow_mount_anyfile, true)
 
 attribute_role mount_roles;
 roleattribute system_r mount_roles;
diff --git a/policy/modules/system/systemd.te b/policy/modules/system/systemd.te
index d837a01..632abf1 100644
--- a/policy/modules/system/systemd.te
+++ b/policy/modules/system/systemd.te
@@ -10,7 +10,7 @@ policy_module(systemd, 1.7.0)
 ## Enable support for systemd-tmpfiles to manage all non-security files.
 ## </p>
 ## </desc>
-gen_tunable(systemd_tmpfiles_manage_all, false)
+gen_tunable(systemd_tmpfiles_manage_all, true)
 
 ## <desc>
 ## <p>
-- 
2.7.4

